src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Column Level Encryption
+summary: SQL users are required to pass encryption keys when reading from or writing to specific columns
+toc: true
+docs_area: reference.sql
+---
+
+CockroachDB v23.2 and later support _column level encryption_. Using this feature, you can encrypt one or more of the columns in each row of a database table.
+
+Column level encryption can be useful for compliance scenarios such as adhering to PCI or GDPR.
+
+Once a column is encrypted, SQL users are required to pass an encryption key when reading from or writing to that column. This prevents users without the encryption key from taking actions like the following:
+
+- Reading sensitive information (like _personally identifiable information_ ([PII](https://en.wikipedia.org/wiki/Personal_data))) when accessing the tables.
+- Adding or updating records without encrypting the specified columns.
+
+{{site.data.alerts.callout_info}}
+The "column-level encryption" feature described on this page specifically refers to either symmetric-key encryption or public-key encryption of data, **not** hashing functions operating on data.
```

Comment:
nit: should we standardize on "column level encryption" vs "column-level encryption"?

Reply:
works for me!  changing this and other instances to remove the hyphen